components/brave_wallet/browser/eth_tx_controller_unittest.cc

```diff
@@ -824,15 +825,16 @@ TEST_F(EthTxControllerUnitTest, ApproveHardwareTransaction) {
   base::RunLoop().RunUntilIdle();
   EXPECT_TRUE(callback_called);
   TestEthTxControllerObserver observer("", "", "", "", std::vector<uint8_t>(),
-                                       mojom::TransactionStatus::Approved);
+                                       mojom::TransactionStatus::Unapproved);
   eth_tx_controller_->AddObserver(observer.GetReceiver());
   callback_called = false;
-  eth_tx_controller_->ApproveHardwareTransaction(
-      tx_meta_id, base::BindLambdaForTesting([&](bool success) {
-        EXPECT_TRUE(success);
+  eth_tx_controller_->GetNonceForHardwareTransaction(
+      tx_meta_id, base::BindLambdaForTesting([&](const std::string& nonce) {
+        EXPECT_TRUE(nonce.size());
```

Comment:
this is usually `EXPECT_FALSE(nonce.empty());`, same applies to the following checks

Reply:
done